docs/analyse_database/README.md

```diff
@@ -0,0 +1,63 @@
+# Analyse database
+The analyse database (gob_analyse) is used by basisinformatie to perform analyses on the GOB data.
+Employees of basisinformatie have direct access to this database. Their access is controlled with
+Postgres roles. This directory contains the necessary SQL scripts. This file explains how to use them.
+
+## Design
+### Schemas and default roles
+All registrations in GOB have their own schemas in the analyse database. There is a bag schema, brk schema,
+gebieden, etc. For each schema there is a corresponding _read role (bag_read, brk_read, gebieden_read,
+etc). This read role has, as the name suggests, read permissions on the schema. There are no write roles
+for these schemas.
+
+There is one extra schema called 'maatwerkbasisinformatie', where (some) basisinformatie employees have
+write permissions. This is their playground where they can place the results of their analyses. Two roles
```

Comment:
Another purpose of this schema is to create views of frequently used queries or joins which can be used by everyone

Reply:
Mentioning the views now too.